src/Compilers/CSharp/Portable/BoundTree/BoundDagEvaluation.cs

```diff
@@ -9,7 +9,7 @@ partial class BoundDagEvaluation
         public override bool Equals(object obj) => obj is BoundDagEvaluation other && this.Equals(other);
         public virtual bool Equals(BoundDagEvaluation other)
```

Comment:
>Equals [](start = 28, length = 6)

Logged #36102 to consider renaming the `Equals` method to something more specific.